documentation/src/test/resources/configuration/http-client-configurer.properties

```
@@ -0,0 +1,2 @@
+# <1>
+hibernate.search.backend.client.configurer = org.hibernate.search.documentation.backend.elasticsearch.client.HttpClientConfigurer
```

Please specify the bean retrieval prefix where possible (here it's `class:`); it's less ambiguous and at some point we'll probably log warnings when it's not used. 

```suggestion
hibernate.search.backend.client.configurer = class:org.hibernate.search.documentation.backend.elasticsearch.client.HttpClientConfigurer
```